app/controllers/admin/admin_invitations_controller.rb

```diff
@@ -42,13 +42,12 @@ def find
       @invitations = @user.invitations if @user
     end
     if !invitation_params[:token].blank?
-      @invitation = Invitation.find_by(token: invitation_params[:token])
+      @invitations = Invitation.where(token: invitation_params[:token]).limit(1)
```

Comment:
Actually, now that we always treat this as a list, I don't think we even need the `.limit(1)` -- just the `where` should do it